frontend/src/features/the-unterstuetzungsfinder/features/the-mail/the-unterstuetzungsfinder-ergebnis.vue

```diff
@@ -240,16 +239,36 @@
             </v-card>
           </v-dialog>
         </v-col>
+        <v-col
```

Comment:
Wäre cool, wenn das alles in eine einzelne Componente kommen könnte. Die Datei ist schon echt groß, das würde es übersichtlicher machen

Reply:
Gute Idee